server_test.go

```diff
@@ -41,3 +56,145 @@ func TestParseHexColor(t *testing.T) {
 		}
 	}
 }
+
+// TestTLSAutoRegeneration creates an expired TLS certificate, to test that a
+// new TLS certificate pair is regenerated when the old pair expires. This is
+// necessary because the pair expires after a little over a year.
+func TestTLSAutoRegeneration(t *testing.T) {
+
+	homePath, err := os.UserHomeDir()
+	if err != nil {
+		t.Fatalf("couldn't retrieve TLS config")
+	}
+	certPath := homePath + "/.lnd/tls.cert"
+	keyPath := homePath + "/.lnd/tls.key"
+
+	expiredCert := genExpiredCertPair(t, certPath, keyPath)
+
+	RPCListener := net.IPAddr{IP: net.ParseIP("127.0.0.1"), Zone: ""}
```

Comment:
preferred variable naming should be camel case, e.g. `rpcListener`, `rpcListeners`, etc